Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/DefaultRiskLevels.kt

```diff
@@ -264,19 +272,19 @@ class DefaultRiskLevels @Inject constructor() : RiskLevels {
             throw NormalizedTimePerDayToRiskLevelMappingMissingException()
```

Comment:
@BMItter I think this should also be gone... is this the one failing test case you're still seeing? Otherwise, we should have a test case that's failing because of this (and it should be part of the JSON thingy)

Reply:
outdated Exceptions have been removed.
(failing test cases this morning had other issues (v2<->v1)), also solved.

Reply:
yes right, you can extend the testcase.json if you want